### ISSUE TITLE
`create_site_academic()` fails to find the downloaded example site

### ISSUE THREAD

**Issue by mikemc** [r-lib/hugodown#89]
``` r
hugodown::create_site_academic()
#> ✔ Writing 'reprex2782c6016ff7e.Rproj'
#> ✔ Adding '.Rproj.user' to '.gitignore'
#> ✔ Downloading academic theme
#> ✔ Copying site components
#> Error: [ENOENT] Failed to search directory '/tmp/RtmpDEzR7l/hugodown278af175a50ec/hugo-academic-4.8.0/exampleSite': no such file or directory
```

<sup>Created on 2020-12-05 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

The problem seems to be that hugodown is looking for a folder called "hugo-academic-4.8.0" but the downloaded folder is actually called "wowchemy-hugo-modules-4.8.0"

**Comment by maelle** [comment]
This is something that should have been fixed by #84, have you tried re-installing hugodown? I hope it helps 🤞

**Comment by mikemc** [comment]
Ah...thank you @maelle I think because the package version has been fixed at "0.0.0.9000" I didn't realize my hugodown version was out of date. Upgrading did indeed fix the problem.
